### PR TITLE
Deserialize pullrequestreviewcommentevent

### PR DESCRIPTION
**Pull request by demaskey**
added some logic to Deserialize a Pull Request Review Comment Event.

### CODE REVIEW

**Comment by demaskey** [comment]
Comment on the conversation tab.

File: sam-app/src/JiraGitHubIntegration/Comment.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+using Newtonsoft.Json;
+using System;
+
+namespace JiraGitHubIntegration
+{
+    public class Comment
+    {
```
Comment (demaskey) [inline sam-app/src/JiraGitHubIntegration/Comment.cs:7]: Starting a review.

**Review by demaskey — COMMENTED** [review]
Submitting a review.

File: sam-app/src/JiraGitHubIntegration/Event.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+using Newtonsoft.Json;
+
+namespace JiraGitHubIntegration
+{
+    public class Event
```
Comment (demaskey) [inline sam-app/src/JiraGitHubIntegration/Event.cs:6]: Type something in

**Review by joeziemba — DISMISSED** [review]

**Pull request merged**